### ISSUE TITLE
[BUG] Swedish text characters not showing up with 7pt font size

### ISSUE THREAD

**Issue by Gartom** [bitfocus/companion#1316]
**Describe the bug**
Swedish characters will not show up on buttons when using 7pt font size. All other font sizes work.

**To Reproduce**
1. Launch Companion GUI
2. Select Buttons tab
3. Create a regular button
4. Write Swedish characters "åäö ÅÄÖ" in the "Text" field 
5. Note that text is showing with 24pt sized font
6. Change font size to 7pt
7. Note that the Swedish characters disappeared from button (this also happens with Swedish characters in dynamic variables from module instances)

**Expected behavior**
Correct text printout on button.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox 83.0
 - Companion Version 2.1.2

**Additional context**

**Comment by dnmeid** [comment]
It's not a bug it's a feature. The 7pt font is hand-designed and very tiny. It is quite challenging to squeeze the ASCII letters into that tiny area. So all the non ASCII chars are working only in the larger fonts.

**Comment by Julusian** [comment]
@dnmeid should this really be 'wontfix'?
I would expect the characters to show up in their ascii form instead. so that "åäö ÅÄÖ" will get shown as "aa0 AAO". Dropping characters completely is a bit weird.

**Comment by Gartom** [comment]
I looked into the font directory and saw that 7px fonts seemed to be hand made for this purpose. I created similar fonts for å, ä, ö, Å, Ä, and Ö, using their corresponding ASCII numbers as file names. Could these be integrated?

![132](https://user-images.githubusercontent.com/4034632/103247268-9c238f00-4966-11eb-9d2a-566818bb0b9d.png)
![134](https://user-images.githubusercontent.com/4034632/103247270-9cbc2580-4966-11eb-9d3f-6a74fa5170bb.png)
![142](https://user-images.githubusercontent.com/4034632/103247271-9cbc2580-4966-11eb-827f-dbee29d28c9c.png)
![143](https://user-images.githubusercontent.com/4034632/103247272-9d54bc00-4966-11eb-92d7-0eea475c6f14.png)
![148](https://user-images.githubusercontent.com/4034632/103247275-9d54bc00-4966-11eb-823b-82e7b0acaf61.png)
![153](https://user-images.githubusercontent.com/4034632/103247276-9d54bc00-4966-11eb-8729-e8e324aeddec.png)





, see attached files.

**Comment by Gartom** [comment]
@dnmeid, the Swedish characters are available in the ascii table (132, 134, 142, 143, 148 and 153). I created 7px fonts in the same manner as the other fonts in the "font" subdirectory. 

If I attach them here they show up as graphics:
![132](https://user-images.githubusercontent.com/4034632/103247501-6337ea00-4967-11eb-9f04-4b056f71638d.png)
![134](https://user-images.githubusercontent.com/4034632/103247502-63d08080-4967-11eb-87b3-1611f53c242d.png)
![142](https://user-images.githubusercontent.com/4034632/103247504-63d08080-4967-11eb-90ee-6f91dfb9e518.png)
![143](https://user-images.githubusercontent.com/4034632/103247505-64691700-4967-11eb-9c2b-c7383b076170.png)
![148](https://user-images.githubusercontent.com/4034632/103247506-64691700-4967-11eb-908d-4cb83fc7fce4.png)
![153](https://user-images.githubusercontent.com/4034632/103247507-64691700-4967-11eb-9f19-2f198c3bbd17.png)

**Comment by Gartom** [comment]
> 
> 
> @dnmeid should this really be 'wontfix'?
> I would expect the characters to show up in their ascii form instead. so that "åäö ÅÄÖ" will get shown as "aa0 AAO". Dropping characters completely is a bit weird.

@Julusian, if it is not possible to integrate the fonts I made, for some reason, it would be better to let "åäö ÅÄÖ" show as "aa**o** AAO".

**Comment by Gartom** [comment]
@dnmeid and @Julusian, I made the Swedish 7px fonts available here: http://gofile.me/2o5tV/QrvlZPzI0

**Comment by Gartom** [comment]
@Julusian and @dnmeid, are you still considering this issue as 'wontfix'?

The 7px font size is useless for Swedish in its current form. 

I will try to fix this myself just because the fun of it. My skills in programming are a bit rusty but as this is a very limited issue to work with it may be a good issue to start with. :-)

I also realized that I probably used the wrong variation of the extended ascii table when adding my fonts, therefore the zip archive has been updated using Latin-1 definition of extended ascii.
